<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleQuery</class>
 <widget class="QWidget" name="VehicleQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>380</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>航班/列车/汽车信息查询 - 低风险旅行模拟系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="COVID_19_Travel_System.qrc">
    <normaloff>:/COVID_19_Travel_System/Resources/Search.png</normaloff>:/COVID_19_Travel_System/Resources/Search.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButtonYes">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>21</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>始发站</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSrcID">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>21</y>
     <width>69</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDstID">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>21</y>
     <width>69</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>21</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>终到站</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidgetSchedule">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>521</width>
     <height>241</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="headerVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="headerShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>类型</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>编号</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>始发站</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>终到站</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>开点</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>到点</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>运行时间</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <zorder>treeWidgetSchedule</zorder>
  <zorder>pushButtonYes</zorder>
  <zorder>label</zorder>
  <zorder>comboBoxSrcID</zorder>
  <zorder>comboBoxDstID</zorder>
  <zorder>label_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="COVID_19_Travel_System.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonYes</sender>
   <signal>clicked()</signal>
   <receiver>VehicleQuery</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
